.travis.yml

```diff
@@ -46,6 +46,13 @@ jobs:
     - name: storybook-smoke-test (node@production)
       <<: *node-prod
       script: yarn storybook-smoke-test
+    - name: yarn serve (node@production)
+      <<: *node-prod
+      script:
+        # `yarn build` takes some time so let's simulate it instead.
+        - mkdir build && touch build/index.html
+        - yarn serve &
+        - sleep 120 && pkill -f node
```

Comment:
Also, `pkill -f node` seems a bit brutal. Are you sure that will be safe? Is every job guaranteed to get its own VM? Travis doesn't use node for anything? Will it ever?

Reply:
I am not sure I can catch the right PID but I will try to do that.

> Are you sure that will be safe?

Yes.

> Is every job guaranteed to get its own VM?

Yes. (technically not a VM anymore but its own system/container, yes)

> Travis doesn't use node for anything?

Nope.

> Will it ever?

Yes. If it does, then it can potentially break builds.